Xamarin.Forms.Xaml/ApplyPropertiesVisitor.cs

```diff
@@ -26,20 +28,17 @@ public ApplyPropertiesVisitor(HydratationContext context, bool stopOnResourceDic
 			StopOnResourceDictionary = stopOnResourceDictionary;
 		}
 
-		Dictionary<INode, object> Values
-		{
+		Dictionary<INode, object> Values {
```

Comment:
Mentally ignore the style changes. A great sign of intelligence is the capacity to think abstractly